### PR TITLE
community/bareos: Upgrade to 18.2.6

### PR DESCRIPTION
**Pull request by bratkartoffel**
This PR upgrades bareos to the latest release, 18.2.6.
It also includes the fixes from #6951.

* -trayicon application was removed (why included anyways?)
* The database backend-drivers are now split into subpackages (lesser dependencies)
* Remove dependency to perl, not needed
* -webui is now built by the "bareos" aport
* (note) the package currently depends on python2 and doesn't build with python3

### CODE REVIEW

File: community/bareos/APKBUILD [hunk 1]
```diff
@@ -2,93 +2,94 @@
 # Contributor: Francesco Colista <fcolista@alpinelinux.org>
 # Maintainer: Francesco Colista <fcolista@alpinelinux.org>
 pkgname=bareos
-pkgver=17.2.7
-pkgrel=1
+pkgver=18.2.6
+pkgrel=2
```
Comment (andypost) [inline community/bareos/APKBUILD]: I think it should be 0

Comment (bratkartoffel) [reply community/bareos/APKBUILD]: Whoops, forgot to change this. Fixed, thanks.

**Comment by fcolista** [comment]
Thanks for your work on this!
Please check the comments in https://github.com/alpinelinux/aports/commit/15a8ef69c854627fc7c16ecf0ab3a09e7e3311e1


.:Francesco

**Comment by bratkartoffel** [comment]
Thanks fcolista, I've applied your suggestions

**Comment by fcolista** [comment]
> Thanks fcolista, I've applied your suggestions

Thanks @bratkartoffel 
Can you rebase the patch against bareos 17.2.7-r2 ?
I have applied your prevoious patches in commit  37923bd0f6d4e33a3b7de6c23e708787c5d7d3bf
Thanks!
.: Francesco

**Comment by bratkartoffel** [comment]
@fcolista Done, this PR can now be merged again.

**Pull request closed without merging**